packages/workflow/src/internals.ts

```diff
@@ -100,6 +101,7 @@ export const LATEST_INTERNAL_PATCH_NUMBER = 1;
  * Implements handlers for all workflow activation jobs.
  */
 export class Activator implements ActivationHandler {
+  readonly moduleCache = new Map<string, unknown>();
```

Comment:
Comment on this one would be useful, especially given that it is used only in `reusable-vm.ts`... Also, could we reasonably (ie. without going out of our way) to avoid initializing a Map unless we actually needs it? Maybe a lazy getter?

Reply:
I don't think the initialization here is critical perf wise.
I'm sure there are many optimizations we can make before this becomes an issue.

I'll add a comment.